h2o-r/h2o-package/R/import.R

```diff
@@ -351,6 +351,20 @@ h2o.loadModel <- function(path) {
   h2o.getModel(res$model_id$name)
 }
 
+h2o.upload_model <- function(path) {
```

Comment:
please add documentation

Reply:
updated also save_model/saveModel/download_model documentation here, hence closing this one: https://github.com/h2oai/h2o-3/pull/4367/